test/test_muscles.cpp

```diff
@@ -150,6 +150,21 @@ TEST(IdealizedActuator, copy)
         EXPECT_EQ(deepCopyNow.position().insertionInLocal().name(), oldName);
         EXPECT_EQ(deepCopyLater.position().insertionInLocal().name(), oldName);
     }
+    {
+        biorbd::Model model(modelPathForMuscleForce);
+        biorbd::muscles::IdealizedActuator idealizedActuator(
+            model.muscleGroup(muscleGroupForIdealizedActuator).muscle(
+                muscleForIdealizedActuator));
+
+        biorbd::muscles::IdealizedActuator shallowCopy(idealizedActuator);
+        biorbd::muscles::IdealizedActuator deepCopyNow(idealizedActuator.DeepCopy());
+        biorbd::muscles::IdealizedActuator deepCopyLater;
+        deepCopyLater.DeepCopy(idealizedActuator);
+
+        EXPECT_NEAR(idealizedActuator.state().excitation(), shallowCopy.state().excitation(), requiredPrecision);
+        EXPECT_NEAR(idealizedActuator.state().excitation(), deepCopyNow.state().excitation(), requiredPrecision);
+        EXPECT_NEAR(idealizedActuator.state().excitation(), deepCopyLater.state().excitation(), requiredPrecision);
```

Comment:
Please use the actual hard-coded value and not the value returned by the muscle. If we ever change the default value (for example by mistake) we want the test to fail and not accommodate for change. If the change was intended, we will change the test accordingly. So having a variable that holds the intended value declared just prior to these lines is the way to go!

Reply:
So instead of using ''idealized.state().excitation()'' you want me to use 0. ?